### DISCUSSION TITLE
Comfy Ui Lora Stacker For Fooocus

### DISCUSSION THREAD

**Discussion by Vincevinny** [lllyasviel/Fooocus#1727] [Ideas]
so i use comfy ui too and the Lora Stacker node from https://github.com/LucianoCirino/efficiency-nodes-comfyui
is soo damn useful when im messing around for fun sometimes i end up with over 20 loras the max as far as i know in fooocus at the moment is 5 im wondering (without coding knowledge) if you could borrow the code and tweak it for fooocus?

so basically we have a little slider or something that can simply add another lora box under the 5th one and so on.

once again no idea how much work this so just an idea

**Comment by mashb1t** [comment]
The 5 LoRAs are referenced in the code quite a few times and LoRA 6 is used in the background to set the LCM LoRA.
There also is an issue in which a user tried to add a new LoRA, which you might find interesting: https://github.com/lllyasviel/Fooocus/issues/1205

**Reply by Vincevinny** [reply]
huh glad you showed me that before i wasted hours tinkering :D i have to learn python...
